.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+sudo: false
+
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+
+before_install:
+  # We do this conditionally because it saves us some downloading if the
+  # version is the same.
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda  
+
+install:
+  - conda create -q -n testenv --yes python=$TRAVIS_PYTHON_VERSION numpy scipy nose pandas flake8
+  #- pip install --no-deps git+https://github.com/quantopian/zipline
```

Comment:
I followed [this](http://conda.pydata.org/docs/travis.html) when setting up miniconda. Conda is used to successfully install scipy and numpy in travis.